#!/usr/bin/python4

def raise_exception_msg(message=""):
    raise NameError(message)
